src/Login/index.tsx

```diff
@@ -44,6 +44,32 @@ export const LoginScreen = (props: LoginProps) => {
     return setErrorMessage("Your email and your password don't match");
   };
 
+  const login = async (email: string, password: string) => {
+    try {
+      const response = await fetch(`${API_URL}/auth/login`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          username: email,
+          password,
+        }),
+      });
+
+      if (response.status >= 200 && response.status < 300) {
+        return await response.json();
```

Comment:
No need to set the `await`, your function being async, you can return an unresolved promise